apps/api/src/function/src/crud.ts

```diff
@@ -106,6 +106,32 @@ export async function insert(fs: FunctionService, engine: FunctionEngine, fn: Fu
   return fn;
 }
 
+export async function insertSchemaOnly(fs: FunctionService, engine: FunctionEngine, fn: Function) {
```

Comment:
Move this duplicated code into a function and use it for both insert and insertSchemaOnly